resources/views/settings/settings.blade.php

```diff
@@ -0,0 +1,79 @@
+@extends('layouts.main_layout')
+
+@section('head')
+
+    <title>Settings</title>
+
+@endsection
+
+@section('mainBody')
+
+    <main class="primary-main row">
+        <section class="grid register-main">
+
+            <form class="form-horizontal">
```

Comment:
Just wanted to add if you make it a POST method you need to include {!! csrf_field() !!} within the form or else you will receive an error